spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/secretmanager/SecretManagerPropertyIdentifier.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2017-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.gcp.autoconfigure.secretmanager;
+
+import org.springframework.cloud.gcp.core.GcpProjectIdProvider;
+import org.springframework.util.Assert;
+
+/**
+ * Identifiers for specifying Secret Manager secrets using (project-id, secret-id, version).
+ *
+ * @author Daniel Zou
+ * @since 1.2.3
+ */
+public class SecretManagerPropertyIdentifier {
```

Comment:
I'm not sure you really need this class. Can you just use the existing `SecretVersionName`?
I understand you need a place to put the `parseFromProperty` method somewhere, but maybe for now you can just put it as a static method in `SecretManagerAccessPropertySource`?

Reply:
Good call, done.